Source/Diagnostics/BTDiagnostics.cpp

```diff
@@ -897,7 +897,6 @@ BTDiagnostics::DefineFieldBufferMultiFab (const int i_buffer, const int lev)
     m_buffer_box[i_buffer].setSmall( m_moving_window_dir, hi_k_lab - m_buffer_size + 1);
     m_buffer_box[i_buffer].setBig( m_moving_window_dir, hi_k_lab );
     amrex::BoxArray buffer_ba( m_buffer_box[i_buffer] );
-    buffer_ba.maxSize(m_max_box_size);
```

Comment:
Should we then remove `m_max_box_size` from the member variables of this class?

Reply:
Seems to be used elsewhere as well:
```
$ ag m_max_box_size
Source/Diagnostics/BTDiagnostics.H
178:    int m_max_box_size = 256;

Source/Diagnostics/ComputeDiagFunctors/BackTransformFunctor.cpp
166:    m_max_box_size = max_box_size;

Source/Diagnostics/ComputeDiagFunctors/BackTransformFunctor.H
127:    int m_max_box_size = 256;

Source/Diagnostics/BTDiagnostics.cpp
263:        if(m_max_box_size < m_buffer_size) m_max_box_size = m_buffer_size;
850:                                             k_index_zlab(i_buffer, lev), m_max_box_size,
```

Reply:
yes we can remove all references to `m_max_box_size`

Reply:
@RemiLehe this is now fixed